src/replication.c

```diff
@@ -1244,11 +1244,12 @@ void syncCommand(client *c) {
  * the primary can accurately lists replicas and their listening ports in the
  * INFO output.
  *
- * - capa <eof|psync2|dual-channel>
+ * - capa <eof|psync2|dual-channel|disable_sync_crc>
```

Comment:
I am not super fond of the disable_sync_crc capability name. maybe a better name woulod be bypass_crc?